src/app.jsx

```diff
@@ -229,6 +229,17 @@ class AppActive extends React.Component {
                                      }
                                      icon={ExclamationCircleIcon} />
                 );
+            } else if (vm.createInProgress) {
+                return (
+                    <EmptyStatePanel title={ cockpit.format(_("Creating VM $0"), cockpit.location.options.name, cockpit.location.options.connection) }
```

Comment:
Arguments don't match up -- template just has `$0` and no `$1`, but you supply two values.